test/jdk/jdk/net/ExtendedSocketOption/DontFragmentTest.java

```diff
@@ -58,6 +58,9 @@ public static void main(String[] args) throws IOException {
     public static void testDatagramChannel() throws IOException {
         try (DatagramChannel c1 = DatagramChannel.open()) {
 
+            if (!c1.supportedOptions().contains(IP_DONTFRAGMENT)) {
+                return;
```

Comment:
Should we assert that this should only happens on mac here? Same remark for the two other places below...

Reply:
I think that is reasonable.